apps/nextjs/README.md

```diff
@@ -1,3 +1,5 @@
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/fork/github/shopify/hydrogen-ui/tree/main/apps/nextjs?file=pages%2Findex.tsx)
+
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Should we update the text of the rest of this readme?

Reply:
I'm not sure. I think leave it for now and maybe we revisit when there are multiple apps. @gfscott do you have any opinion on the content of this readme?